src/xcode/ENA/ENA/Source/Scenes/RecycleBin/Cells/RecycleBinItemCellModel.swift

```diff
@@ -123,6 +123,18 @@ class RecycleBinItemCellModel {
 		}
 	}()
 
+	lazy var quaternaryInfo: String? = {
+		guard let expirationDate = Calendar.current.date(byAdding: .day, value: 30, to: recycleBinItem.recycledAt) else {
```

Comment:
Could we make the `30` a static let on `RecycleBin` or something like that so we have a single place to change that value if we ever decide to change it?